src/frontend/components/UI/ThemeSelector/index.tsx

```diff
@@ -63,7 +63,6 @@ export const ThemeSelector = () => {
       setThemesPath(config.customThemesPath || '')
       loadThemes()
     }
-
```

Comment:
let's remove these extra changes, please make sure the PR doesn't touch files unrelated to this feature